### PR TITLE
Warn when building with default server.crt file

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
@kacf

### CODE REVIEW

File: recipes-mender/mender/mender_0.1.bb [hunk 1]
```diff
@@ -45,6 +45,11 @@ do_compile() {
   export GOPATH
   PATH="${B}/bin:$PATH"
   export PATH
+  DEFAULT_CERT_MD5="e034532805e44b9125b443c32bbde581"
+
+  if [ $(md5sum $MENDER_CERT_LOCATION | awk '{ print $1 }') = $DEFAULT_CERT_MD5 ]; then
```
Comment (kacf) [inline recipes-mender/mender/mender_0.1.bb]: It's good style to quote what's between the `$(...)` with *double quotes*. That way the logic doesn't break if the `MENDER_CERT_LOCATION` is missing or empty.

Comment (GregorioDiStefano) [inline recipes-mender/mender/mender_0.1.bb]: ah, yes, i usually use shellcheck to remind me about this, but the plugin was turned on for .bb files

**Comment by GregorioDiStefano** [comment]
@kacf updated

**Comment by kacf** [comment]
Thanks, running in Jenkins.

**Comment by GregorioDiStefano** [comment]
For some reason, this isn't working. Will take a look on Monday.

**Comment by GregorioDiStefano** [comment]
Changed and tested locally, testing with PR machine before merging

**Comment by GregorioDiStefano** [comment]
@kacf this is working, so please merge if you deem everything to be okay.

**Pull request merged**